lametro/templates/base.html

```diff
@@ -72,7 +72,7 @@
                             <a href="{% url 'lametro:search' %}">Board Reports</a>
                         </li>
                         <li>
-                            <a href="{% url 'lametro:archive-search' %}">Archive Search</a>
```

Comment:
If we're removing this link, do we need [the archive search page](https://boardagendas.metro.net/archive-search) anymore, or can we remove it from the app?

Reply:
Cc @shrayshray